sdk/storage/storage-common/review/storage-common.api.md

```diff
@@ -0,0 +1,24 @@
+## API Report File for "@azure/storage-common"
+
+> Do not edit this file. It is a report generated by [API Extractor](https://api-extractor.com/).
+
+```ts
+
+import type { HttpClient } from '@azure/core-rest-pipeline';
```

Comment:
If storage-common is released as a separate package, this will be exposed to customer. We only tested this as used in storage package, we cannot guarantee its robust for customer.